tracer/src/Datadog.Trace/Tagging/AspNetTags.cs

```diff
@@ -23,5 +25,8 @@ internal partial class AspNetTags : WebTags
 
         [Tag(Tags.HttpRoute)]
         public string HttpRoute { get; set; }
+
+        [Tag(Trace.Tags.InstrumentationName)]
+        public string InstrumentationName { get; set; } = ComponentName;
```

Comment:
This is used in pretty much every integration. Should we move it to `InstrumentationTags`? (not necessarily this PR, just something to think about)